### PR TITLE
feat: change the text to Hello

### LINKED ISSUE #1: Modify the header test
Change the text description to Hello

### PR DESCRIPTION
**Pull request by SulemanRasheed132446**
- change the test to hello

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Home: NextPage = () => {
 
       <main className={styles.main}>
         <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
+          Welcome to <a href="https://nextjs.org">Hello</a>
```
Comment (skeithc) [inline pages/index.tsx:17]: Maybe we should add an exclamation point here

Comment (skeithc) [inline pages/index.tsx]: We should put an exclamation point here

**Review by skeithc — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (SulemanRasheed132446) [reply pages/index.tsx]: @skeithc I have made the changes for the excalamation mark

**Review by skeithc — APPROVED** [review]
lgtm!

**Pull request merged**